mscore/shortcutcapturedialog.cpp

```diff
@@ -116,11 +114,8 @@ void ShortcutCaptureDialog::keyPress(QKeyEvent* e)
 
       k += e->modifiers();
       // remove shift-modifier for keys that don't need it: letters and special keys
-      if ((k & Qt::ShiftModifier) && ((e->key() < 0x41) || (e->key() > 0x5a) || (e->key() >= 0x01000000))) {
-            qDebug() << k;
-      	k -= Qt::ShiftModifier;
-            qDebug() << k;
-            }
+      if ((k & Qt::ShiftModifier) && ((e->key() < 0x41) || (e->key() > 0x5a) || (e->key() >= 0x01000000)))
+            k -= Qt::ShiftModifier;
```

Comment:
This debug message is not obsolete and is used in debugging process of the events.